elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -375,6 +376,15 @@ public String getDBPassword(DBConfig config) {
         };
     }
 
+    /**
+     * Provides the default SQLDialect type.
+     *
+     * @return {@link SQLDialect} type for default DataSource Object.
+     */
+    default String getDefaultDialect() {
+        return "MYSQL";
```

Comment:
use one of the supported types.

Reply:
wanted to ensure override works properly.

Reply:
changed to Hive.